### PR TITLE
Feat:Create iOS app development landing section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Description:**
Create landing section.
Android app page fixes:
1. Fix margin-top case study (mobile).
2. Fix touch not work in success story(mobile).
3. Remove auto scroll deom development section.
4. Add z-index in header.

**Mobile performance**
image

**Mobile Image:**
image
**Desktop Image**
image

 **Video:**
[screen-capture (2).webm](https://github.com/canopas/canopas-website/assets/98794636/e17a6d17-7068-4583-bf5d-51c86eec0ef3)

### CODE REVIEW

**Review by cp-mansi-d — APPROVED** [review]

File: vue-frontend/src/components/ios-app-development/LandingSection.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <div
+          class="tw--mt-[2.7rem] tw-mr-28 xs:tw--mt-14 xs:tw-mr-40 md:tw-hidden"
+        >
+          <router-link
+            class="gradient-btn consultation-btn tw-h-[3.3rem] tw-w-max tw-items-left tw-rounded-full"
+            to="/contact"
+            @click.native="mixpanel.track('tap_ios_development_cta')"
+          >
+            <span
+              class="tw-font-inter-semibold tw-text-[1rem] tw-font-normal tw-leading-[1.21rem] !tw-tracking-[0] md:tw-text-[1.09375rem] md:tw-text-[1.1875rem] md:tw-leading-[1.3125rem] md:tw-leading-[1.4375rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.4375rem]"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/ios-app-development/LandingSection.vue:65]: Keep line height same as figma currently text are not looking center

File: vue-frontend/src/components/ios-app-development/LandingSection.vue [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          >
+        </router-link>
+      </div>
+      <div
+        class="tw-flex tw-flex-col md:tw-flex-row tw-items-center tw-justify-center md:tw-flex-1"
+      >
+        <img
+          :src="landing400"
+          :srcset="`${landing400} 400w,${landing800} 800w`"
+          class="tw-mt-7 tw-overflow-visible md:tw-mt-0 tw-h-[21.49113rem] xs:tw-h-[24.49113rem] md:tw-hidden tw-w-[23.77063rem] tw-object-cover"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/ios-app-development/LandingSection.vue:38]: In some devices this image line is not touching in center of button

File: vue-frontend/src/components/ios-app-development/LandingSection.vue [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  >
+    <div class="tw-container tw-p-4 tw-text-left md:tw-flex xl:tw-my-8">
+      <div class="md:tw-flex-1 lg:tw-mt-16">
+        <h1
+          class="tracking-[0.08rem] tw-pr-2 tw-mb-2 sm:tw-mb-0 tw-font-inter-semibold md:tw-font-inter-bold md:tw-text-[2.75rem] xl:tw-text-[3.75rem] tw-text-[2rem] md:tw-leading-[3.625rem] xl:tw-leading-[5.625rem] tw-leading-[2.6rem] tw-text-black-core/[0.87] md:tw-font-inter-bold"
+        >
+          iOS App Development Company
+        </h1>
+        <p
+          class="tw-font-inter-medium md:tw-font-inter-regular tw-pt-2 xl:tw-text-[1.25rem] tw-text tw-text-[1rem] xl:tw-leading-[1.875rem] 2xl:tw-w-4/5 tw-leading-[1.5rem] tw-text-black-core/[0.87]"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/ios-app-development/LandingSection.vue:13]: Use default class here for tw-leading-[1.5rem]

File: vue-frontend/src/components/ios-app-development/LandingSection.vue [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+          :src="landingmd800"
+          :srcset="`${landingmd800} 800w`"
+          class="tw-overflow-visible tw-hidden md:tw-block tw-h-full xl:!tw-h-full lg:!tw-mt-[10%] lg:!tw-h-[98%] tw--ml-[82%] lg:tw--ml-[52%] xl:tw--ml-[70%] tw-w-[55%] tw-object-cover"
+          alt="ios-landing"
+        />
+        <img
+          v-else
+          :src="landing1200"
+          :srcset="`${landing1200} 1200w`"
+          class="tw-overflow-visible tw-hidden md:tw-block tw-h-[105%] tw-mt-8 xl:tw--ml-[70%] tw-w-[55%] tw-object-cover"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/ios-app-development/LandingSection.vue:52]: it's not feasible to use percentage value greater than 100%

**Review by cp-mansi-d — APPROVED** [review]

File: vue-frontend/src/components/ios-app-development/LandingSection.vue (hunk 2 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/ios-app-development/LandingSection.vue:38]: Its aligned.Try to refresh and check.

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**